src/main/java/no/ssb/dlp/pseudo/service/pseudo/PseudoServiceTypeConverterRegistrar.java

```diff
@@ -0,0 +1,27 @@
+package no.ssb.dlp.pseudo.service.pseudo;
+
+import io.micronaut.core.convert.MutableConversionService;
+import io.micronaut.core.convert.TypeConverterRegistrar;
+import jakarta.inject.Singleton;
+import no.ssb.dlp.pseudo.core.PseudoSecret;
+import no.ssb.dlp.pseudo.core.func.PseudoFuncRule;
+import no.ssb.dlp.pseudo.core.typeconverter.PseudoFuncRuleTypeConverter;
+import no.ssb.dlp.pseudo.core.typeconverter.PseudoSecretTypeConverter;
+
+import java.util.Map;
+
+/**
+ *  Micronaut {@link io.micronaut.core.convert.TypeConverter} implementations defined in a different
+ *  package like the ones from {@link no.ssb.dlp.pseudo.core} need to be manually registered
+ *  using this {@link io.micronaut.core.convert.TypeConverterRegistrar} implementation.
+ *
+ * @author Nicholas Jaunsen
```

Comment:
We should not use @author tags as they promote/signals personal ownership in code. Information is already available in git.

We should probably have an ADR for this.

Reply:
What's an ADR?